app/templates/_package.json

```diff
@@ -4,14 +4,10 @@
   "main": "<%- main %>",
   "private": true,
   "scripts": {
-<% if (proj_type === 'es6') { %>
-    "build:ios": "BABEL_ENV=ios npm run build",
-    "build:android": "BABEL_ENV=android npm run build",
-    "build:win": "BABEL_ENV=win npm run build",
-    "build": "rm -rf dist/ && babel --compact false --out-dir dist/ src/",
-<% } else if (proj_type === 'ts') { %>
-    "build": "tsc -p .",
+<% if (proj_type === 'ts') { %>
+  "build": "tsc -p ."
+<% } else  { %>
+  "start": "tabris serve"
```

Comment:
Hmm, yes I suppose it should. I'm not at all familiar with Type Script. 

So we would need build and start I imagine?